.npmignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
should be same a git ignore except the dist

Reply:
Done